src/core/annotation.js

```diff
@@ -973,8 +1033,15 @@ class PopupAnnotation extends Annotation {
     let parentSubtype = parentItem.get('Subtype');
     this.data.parentType = isName(parentSubtype) ? parentSubtype.name : null;
     this.data.parentId = dict.getRaw('Parent').toString();
-    this.data.title = stringToPDFString(parentItem.get('T') || '');
-    this.data.contents = stringToPDFString(parentItem.get('Contents') || '');
+
+    if (parentItem.has('RT')) {
+      if ((parentItem.get('RT').name ||
```

Comment:
Let's simplify this to just `if (parentItem.get('RT').name === AnnotationReplyType.GROUP) {` because `RT` must always exist because of the `has` check above.